### PR TITLE
Workaround for PSX DESR

### PR DESCRIPTION
**Pull request by AKuHAK**

### CODE REVIEW

File: ee/gs/src/gsFontM.c [hunk 1]
```diff
@@ -157,7 +157,12 @@ int gsKit_fontm_unpack(GSFONTM *gsFontM)
 	if(FontMFile < 0)
 	{
 		printf("Failed to open FONTM from ROM0\n");
-		return -1;
+		FontMFile = open("rom0:FONTM", O_RDONLY);
+		if(FontMFile < 0)
+		{
+			FontMFile = open("mc0:FONTM", O_RDONLY);
```
Comment (israpps) [inline ee/gs/src/gsFontM.c:163]: Wouldn't it be better to use a relative path?

Just saying

Comment (AKuHAK) [reply ee/gs/src/gsFontM.c:163]: It is not safe to rely on CWD inside library as current working dir can randomly change during library use but call to the FONTM should remain intact.

**Comment by sp193** [comment]
Given that this file technically belongs to the OSD (which the PSX does not have, thus the file is absent), why not consider the use of another font library like FreeType? You would be also free to choose whatever font you want, which is also required to enable the use of additional languages.

Is the aim purely about making the fontm library work on more hardware?

**Comment by AKuHAK** [comment]
Currently the aim is to provide at least something for DESR users to get rid of crashing applications, its pure workaround for projects that heavily rely on FONTM usage. But yes , seeing open source alternative for FONTM would be nice.

P.S. made a mistkae, will try to fix it today.

**Comment by uyjulian** [comment]
KROM could be another alternative font file, and is also on DESR and PS3 PS2 emulator

**Comment by sp193** [comment]
At some point, I used KROM. It is ugly and has a different format from FONTM.

If code changes can be made, I would suggest not coupling the app with some board-specific file.

I believe that AKuHAK's motivation is to provide a drop-in solution for apps that were made to use FONTM, but there is no desire to rework said apps to use a different font-rendering library.

File: ee/gs/src/gsFontM.c (hunk 1 above)
Comment (AKuHAK) [reply ee/gs/src/gsFontM.c:163]: oh, totally forgot about closing

**Pull request closed without merging**